torch/csrc/jit/serialization/import_source.cpp

```diff
@@ -367,13 +367,22 @@ struct SourceImporterImpl : public Resolver,
         c10::QualifiedName(qualified_classname), cu_, is_module);
 
     std::vector<Def> methods;
-    std::vector<ResolverPtr> resolvers;
+    std::vector<ResolverPtr> method_resolvers;
+    std::map<std::string, Def> pre_hook_def_map;
+    std::map<std::string, Def> hook_def_map;
+    std::map<std::string, ResolverPtr> pre_hook_resolver_map;
+    std::map<std::string, ResolverPtr> hook_resolver_map;
```

Comment:
Do these maps need to be ordered?

Reply:
no because the ordering is gotten from the hook_names/pre_hook_names vectors